dumb-jump.el

```diff
@@ -758,6 +758,23 @@ or most optimal searcher."
            :regex "object\\s*JJJ\\s*\\\(?"
            :tests ("object test(object)"))
 
+    ;; solidity
+    (:type "function" :supports ("ag" "grep" "rg" "git-grep") :language "solidity"
+           :regex  "function\\s*JJJ\\s*\\\("
+           :tests ("function test() internal" "function test (uint x)" "function test() external"))
+
+    (:type "modifier" :supports ("ag" "grep" "rg" "git-grep") :language "solidity"
+           :regex  "modifier\\s*JJJ\\s*\\\("
+           :tests ("modifier test()" "modifier test ()"))
+
+    (:type "event" :supports ("ag" "grep" "rg" "git-grep") :language "solidity"
+           :regex  "event\\s*JJJ\\s*\\\("
+           :tests ("event test();" "event test (uint indexed x)" "function test(uint x, address y)"))
```

Comment:
Tests are failing on `function test(uint x, address y)`, which makes sense as this `regex` is only looking for `event`